### PR TITLE
Merge bazel filetypes and update to include MODULE.bazel

### PR DESCRIPTION
**Pull request by dapirian**

### CODE REVIEW

**Review by TylerJang27 — APPROVED** [review]
LGTM. Do you know if there was a historical reason why we separated them out?

**Comment by dapirian** [comment]
> LGTM. Do you know if there was a historical reason why we separated them out?

Pedanticness. I ended up leaving the separation for backwards compat but used the `inherits` feature to consolidate them forward-looking.

**Review by det — APPROVED** [review]

File: yada.pyi
Comment (TylerJang27) [inline yada.pyi:1]: Remove plz

Comment (dapirian) [reply yada.pyi:1]: oops thanks

**Pull request merged**